Dockerfile

```diff
@@ -87,6 +100,7 @@ CMD ["/htapps/babel/imgsrv/bin/startup_imgsrv"]
 FROM babel-base AS apache
 
 RUN apt-get -y install apache2 libapache2-mod-fcgid
+RUN apt-get install -y nodejs npm
```

Comment:
It might make more sense for this to be a separate image that just does the JS stuff and stores artifacts in a directory that can be shared with apache; that would also help with being able to provide what https://github.com/hathitrust/catalog/blob/main/sys/Interface.php#L78 needs to find & serve assets locally rather than via netlify

Reply:
In which case, we could use the npm image rather than installing whatever npm debian happens to be able to provide